README.md

```diff
@@ -1,7 +1,7 @@
 # aerospike-kubernetes-enterprise
 
-This project contains the init container used in Kubernetes (k8s) and the Aerospike StatefulSet definition.
-This manifest will allow you to deploy a fully formed Aerospike cluster in minutes.
+DEPRECATED: This Helm chart is no longer actively developed, instead use (https://artifacthub.io/packages/helm/aerospike/aerospike-kubernetes-operator)
```

Comment:
Let's not make it deprecated. This is the main README and this holds our operator helm charts as well. 

Update this README instead https://github.com/aerospike/aerospike-kubernetes-enterprise/tree/master/helm

Update message as follow

## DEPRECATED
This Helm chart is no longer actively developed, instead use (https://artifacthub.io/packages/helm/aerospike/aerospike-kubernetes-operator)